go.sum

```diff
@@ -90,6 +90,8 @@ github.com/Nvveen/Gotty v0.0.0-20120604004816-cd527374f1e5 h1:TngWCqHvy9oXAN6lEV
 github.com/Nvveen/Gotty v0.0.0-20120604004816-cd527374f1e5/go.mod h1:lmUJ/7eu/Q8D7ML55dXQrVaamCz2vxCfdQBasLZfHKk=
```

Comment:
@faisalnaveedRNS I have seen that pkcs github libraries are replaced with our implementation, so that's good. 
How about github.com/Nveen/Gotty as well as github.com/OneOfOne/xxhash? 
Do you guys feel comfortable using these? They are not indirect like the ones above. 
Nveen/Gotty is for interacting with terminal, but couldn't we have used our own implementation for the XXHash? 
Let me know if we can discuss this further before approving this PR.